### PR TITLE
Automated PR for APL-1.0

### PR DESCRIPTION
**Pull request by myndzi**

### CODE REVIEW

File: src/APL-1.0.xml [hunk 1]
```diff
@@ -0,0 +1,970 @@
...
+         the specific adaptive features applicable to this License. For example, without limiting the
+         foregoing, (a) for selected choice of law and jurisdiction see Part 3 of Exhibit A; (b) for the
+         selected definition of Third Party see Part 4 of Exhibit A; and (c) for selected patent licensing
+         terms (if any) see Section 2.2 below and Part 6 of Exhibit A.</p>
+      <list>
+        <li>
+          <b>1.</b>
+          <p>DEFINITIONS.</p>
+        </li>
+        <li>
```
Comment (sam-ellis) [inline src/APL-1.0.xml]: Consider starting a new \<list> here, since strictly 1.1, 1.2 etc are children of 1.

File: src/APL-1.0.xml [hunk 2]
```diff
@@ -0,0 +1,970 @@
...
+          <b>1.20.</b>
+          <p>"THIRD PARTY" has the meaning assigned in Part 4 of Exhibit A.</p>
+        </li>
+        <li>
+          <b>2.</b>
+          <p>LICENSE.</p>
+        </li>
+        <li>
```
Comment (sam-ellis) [inline src/APL-1.0.xml]: Consider starting new list of 2.1, 2.2 etc since they are children of 2.

File: src/APL-1.0.xml [hunk 3]
```diff
@@ -0,0 +1,970 @@
...
+                 patents, trade secrets or any other intellectual property of Initial Contributor,
+                 Subsequent Contributor, or Distributor except as expressly stated herein. 
+          </p>
+        </li>
+        <li>
+          <b>3.</b>
+          <p>DISTRIBUTION OBLIGATIONS.</p>
+        </li>
+        <li>
+          <b>3.1.</b>
```
Comment (sam-ellis) [inline src/APL-1.0.xml]: Consider starting new list for 3.1, 3.2 etc since they are children of 3.

File: src/APL-1.0.xml [hunk 4]
```diff
@@ -0,0 +1,970 @@
...
+             the case may be. Any provision in a Supplement File that otherwise purports to modify, vary,
+             nullify or amend any right, obligation or representation contained herein shall be deemed void
+             to that extent, and shall be of no force or effect.</p>
+        </li>
+        <li>
+          <b>4.</b>
+          <p>COMMERCIAL USE AND INDEMNITY.</p>
+        </li>
+        <li>
+          <b>4.1.</b>
```
Comment (sam-ellis) [inline src/APL-1.0.xml]: Consider starting new list for 4.1, 4.2 etc since they are children of 4.

**Comment by bradleeedmondson** [comment]
not legal XML yet, hold off on merging

**Comment by jlovejoy** [comment]
@bradleeedmondson - I thought I put a note, but don't see it now. I didn't get through the whole license and committed partial review of nesting list tags, as I'm on sketchy connection and didn't want to lose what I'd done so far!

**Comment by bradleeedmondson** [comment]
Oh fair enough -- I didn't look for that kind of note, and I've done the
same thing myself. I just wanted to give this a look over since you were
merging other PRs.

All good!

Thx,
Brad

>

**Comment by jlovejoy** [comment]
fixed rest of list tags - but still getting errors at last three end tags.  @sam-ellis or @goneall  could one of you have a fresh look?

**Comment by zvr** [comment]
the 'p' start tag on 831 does not close; I'm on it

**Pull request merged**